src/styles/index.scss

```diff
@@ -1,11 +1,23 @@
 $bluecolor: #b7daf72d;
-
+$nav-red: #ff0000;
+$nav-item-black: #3a3a3a;
 body {
   background: $bluecolor;
   margin: 0;
   padding: 0;
 }
 
+header {
+  nav .nav-item {
+    color: $nav-item-black;
+    margin: 0 0.5em;
+    text-decoration: none;
+  } // nav .nav-item
```

Comment:
Are the commented lines here being used?

Reply:
@mbilesanmi Test snap files removed